docusaurus.config.js

```diff
@@ -95,11 +95,6 @@ module.exports = {
 
       copyright: `<p style="text-align:left;">Open-sourced with Apache 2.0 License.</p>`,
     },
-    algolia: {
```

Comment:
Why removed? dont we need it for search?

Reply:
good catch! I was doing some tests and which I shouldn't commit this line